groups/cvo2/profiles/heritage-staging-eu-west-2/vars

```diff
@@ -0,0 +1,72 @@
+aws_region  = "eu-west-2"
+aws_account = "heritage-staging"
+
+# shorthand
+account = "hstg"
+region  = "euw2"
+
+# NetApp Cloud Volumes Ontap (CVO)
+cvo2_name                    = "netapp-new"
+cvo_instance_type           = "m5.4xlarge"
+cvo_license_type            = "ha-cot-premium-paygo"
+cvo_is_ha                   = true
+connector_account_access_id = "CloudProviderAccount-AwVeHvwJ"
+cvo_instance_ami_id         = "ami-0f3eef449fe9501e4"
+
+cvo_floating_ips            = [
+    "192.168.255.17",
+    "192.168.255.18",
+    "192.168.255.19",
+    "192.168.255.20"
+]
```

Comment:
https://github.com/companieshouse/netapp-terraform/pull/75/files#r1315813785

Also have these IPs been verified / checked? If not, do we want this profile at this stage?